Makefile

```diff
@@ -67,14 +67,21 @@ post-install:
 	@# Must remove the .babelrc for 0.42.0 to work correctly
 	@# Need to copy custom ImagePickerModule.java that implements correct permission checks for android
 	@rm node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
-	@cp ./ImagePickerModule.java node_modules/react-native-image-picker/android/src/main/java/com/imagepicker
+	@cp ./native_modules/ImagePickerModule.java node_modules/react-native-image-picker/android/src/main/java/com/imagepicker
+	@# Add caching support for native module constants
+	@rm node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java
+	@rm node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob/RNFetchBlob.java
+	@cp ./native_modules/RNDeviceModule.java node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo
+	@cp ./native_modules/RNFetchBlob.java node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob
```

Comment:
this one worries me as we are going to be using a fork of this lib with support for custom stuff that we are adding, perhaps we can add your changes on this file there

Reply:
Not a problem.  Which native module are you planning on forking?  `react-native-device-info` or `react-native-fetch-blob`?

We should be able to change the files in the fork, that would make things easy to maintain